src/main/java/core/basesyntax/service/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;

```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package declaration to adhere to Java style guidelines.